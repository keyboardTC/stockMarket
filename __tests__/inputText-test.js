
import { TextInput } from 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';


// Testing  Input component
it('renders the TextInput component', () => {
    const tree = renderer
        .create(<TextInput autoCorrect={false} value="apple banana kiwi" />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
