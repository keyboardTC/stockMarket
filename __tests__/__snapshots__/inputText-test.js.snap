// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the TextInput component 1`] = `
<TextInput
  autoCorrect={false}
  value="apple banana kiwi"
/>
`;
